modules/tinymce/src/plugins/table/main/ts/api/Options.ts

```diff
@@ -144,6 +155,13 @@ const getDefaultStyles = (editor: Editor): Record<string, string> => {
   return options.isSet('table_default_styles') ? defaultStyles : determineDefaultStyles(editor, defaultStyles);
 };
 
+const getDefaultAttributes = (editor: Editor): Record<string, string> => {
+  // Note: The we don't rely on the default here as we need to dynamically lookup the widths based on the current editor state
+  const options = editor.options;
+  const defaultStyles = options.get('table_default_attributes');
+  return options.isSet('table_default_attributes') ? defaultStyles : determineDefaultAttributes(editor, defaultStyles);
```

Comment:
I think this would be clearer?
```suggestion
  const defaultAttributes = options.get('table_default_attributes');
  return options.isSet('table_default_attributes') ? defaultAttributes : determineDefaultAttributes(editor, defaultAttributes);
```

Reply:
Looks like the wrong variables were changed in https://github.com/tinymce/tinymce/pull/7908/commits/37aa3595b8d5e75ad5752704097b83cbc34cd3e6

- [ ] Fix incorrect variable names